packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -1,6 +1,9 @@
 import React from "react";
 import PropTypes from "prop-types";
 import RawButton from "@paprika/raw-button";
+import CaretDownIcon from "@paprika/icon/lib/CaretDown";
+import CaretUpIcon from "@paprika/icon/lib/CaretUp";
+import TimesCircleIcon from "@paprika/icon/lib/TimesCircle";
```

Comment:
What do you think about importing this icons with names that describe their usage?  Like `ClearIcon` vs `TimesCircleIcon`?  Or `ExpandIcon` vs `CaretDownIcon`?

Reply:
I think keeping the names same everywhere is easier for other devs who looks into the code and want to find out what icons are used, also can be easily copied and pasted. But I'm ok with both!